storage/store_metrics.go

```diff
@@ -74,9 +75,91 @@ var (
 	metaRangeSnapshotsPreemptiveApplied = metric.Metadata{Name: "range.snapshots.preemptive-applied"}
 
 	// Raft processing metrics.
-	metaRaftSelectDurationNanos  = metric.Metadata{Name: "process-raft.waitingnanos"}
-	metaRaftWorkingDurationNanos = metric.Metadata{Name: "process-raft.workingnanos"}
-	metaRaftTickingDurationNanos = metric.Metadata{Name: "process-raft.tickingnanos"}
+	metaRaftSelectDurationNanos = metric.Metadata{Name: "process-raft.waitingnanos",
+		Help: "Nanoseconds spent in store.processRaft() waiting"}
```

Comment:
optional: if you feel like it, could you rename the `process-raft.*nanos` metrics to `raft.*.ns`
you'd have to adjust the UI as well.